Chatkit.xcodeproj/project.pbxproj

```diff
@@ -407,6 +435,16 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		737E3F8023EAC9E000B50837 /* Frameworks */ = {
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				737E3F9523EACACB00B50837 /* PusherChatkit.framework in Frameworks */,
+				737E3F9423EACACB00B50837 /* Environment.framework in Frameworks */,
```

Comment:
Why do we need `Environment` in `Functional Tests`?

Reply:
Good question.  Its not required, I've removed it. (I had copied the `Unit Tests` setup. The `Unit Tests` needs it because it actually performs some tests on the content of `Environment`.  `Functional Tests` do not.)